packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/offers/translations/Messages_fr_FR.json

```
@@ -11,5 +11,6 @@
   "dedicatedCloud_offer_upgrade_choice_backup_time": "Choix de l’heure de début des sauvegardes",
   "dedicatedCloud_offer_upgrade_long_term_retention": "11 points de rétention longue durée (GPS rétention)",
   "dedicatedCloud_offer_upgrade_proxy_backup": "1 backup proxy par hôte",
-  "dedicatedCloud_offer_current_offer": "Offre actuelle"
+  "dedicatedCloud_offer_current_offer": "Offre actuelle",
+  "dedicatedCloud_offer_price_per_vm": "/VM"
```

```suggestion
  "dedicatedCloud_offer_price_per_vm": "{{price}}/VM"
```
or else the translators won't understand the context :)

---

But the price is being displayed by the `catalog-price` component from https://github.com/ovh/manager/tree/develop/packages/manager/modules/catalog-price